ember-cli-build.js

```diff
@@ -1,9 +1,13 @@
 'use strict';
 
+const nodeSASS = require('node-sass');
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
   let app = new EmberAddon(defaults, {
+    sassOptions: {
```

Comment:
Is the default `ember-cli-sass` engine not node-sass?

Reply:
Not any more. Starting with v8, the blueprint adds a pure-JS `sass` dependency to the consuming project and uses that by default unless specified otherwise. If we want to stick with node-sass, I believe we need to add that as a dependency to this project before requiring it here.